src/components/Shared/Header/Header.css

```diff
@@ -2,21 +2,85 @@
 	position: fixed;
```

Comment:
The header doesn't seem to scale very well on mobile... 

![image](https://user-images.githubusercontent.com/10137206/63187144-272f0e00-c02c-11e9-9a7f-0abd448470e5.png)

Reply:
Maybe remove all of the text when you get to the smaller screen size, and just have the icons?